src/stylesheets/components/_wheel-select.scss

```diff
@@ -0,0 +1,78 @@
+.wheel-select {
+  position: relative;
+
+  width: 100%;
+  margin-top: rem(64px);
+
+  &__selected-zone {
+    position: absolute;
+    top: 35%;
+
+    display: inline-block;
+    width: 100%;
+    height: rem(56px);
+
+    border-top: 0.5px solid map-get($brand-white, 'primary');
+    border-bottom: 0.5px solid map-get($brand-white, 'primary');
+  }
+
+  &__container {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: center;
+    height: rem(55px);
+
+    opacity: 0.5;
+
+    pointer-events: none;
+  }
+
+  &__text {
+    color: map-get($brand-white, 'primary');
+    font-size: map-get($font-sizes, 'lg');
+    line-height: rem(25px);
+    text-align: center;
+  }
+
+  &__text--multiple-select {
+    @include make-col(6);
+
+    text-align: left;
+  }
+
+  &__checkbox-container {
+    @include make-col(6);
+
+    text-align: right;
+  }
+
+  &__checkbox-label {
+    width: rem(30px);
+    height: rem(30px);
+    margin-top: rem(10px);
+
+    border: rem(1px) solid map-get($brand-gray, 'primary');
+    border-radius: 50%;
+    cursor: pointer;
+  }
+
+  &__checkbox {
+    visibility: hidden;
+  }
+
+  &__checkbox:checked + label {
+    background-color: map-get($brand-white, 'primary');
+    background-image: url('../../images/icons/check-icon-white.png');
+  }
+}
+
+/* stylelint-disable */
+.slick-active .wheel-select__element {
```

Comment:
@rafayet-monon As you put in here, you should wrap with this `.wheel-select`

Reply:
Fixed in [ec74979](https://github.com/rafayet-monon/surveyor/pull/19/commits/ec74979c88e6b5c56f1088cd13456b4ad5654084)

Reply:
Not like this @rafayet-monon 😅
what I mean it should be like this below
```scss
.wheel-select {
  .slick-active .wheel-select__element {}
}
```
Because you put in the component and then you should wrap with `.wheel-select`

Reply:
Sorry. My bad. Fixed in [ec74979](https://github.com/rafayet-monon/surveyor/pull/19/commits/d27b2f2fd55bb2d83853db5760e2d6cd08d30ab9) @Lahphim